modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbStorage.java

```diff
@@ -398,28 +391,31 @@ public void destroy() {
     }
 
     private byte[] partitionStartPrefix() {
-        return ByteBuffer.allocate(Short.BYTES).order(ByteOrder.BIG_ENDIAN)
-            .putShort((short) (partitionId))
-            .array();
+        return ByteBuffer.allocate(PREFIX_SIZE_BYTES).order(ByteOrder.BIG_ENDIAN)
+                .putInt(tableId)
+                .putShort((short) (partitionId))
+                .array();
     }
 
     private byte[] partitionEndPrefix() {
-        return ByteBuffer.allocate(Short.BYTES).order(ByteOrder.BIG_ENDIAN)
-            .putShort((short) (partitionId + 1))
-            .array();
+        return ByteBuffer.allocate(PREFIX_SIZE_BYTES).order(ByteOrder.BIG_ENDIAN)
+                .putInt(tableId)
+                .putShort((short) (partitionId + 1))
+                .array();
     }
 
     private byte[] txIdToKey(UUID txId) {
-        return ByteBuffer.allocate(Short.BYTES + 2 * Long.BYTES).order(ByteOrder.BIG_ENDIAN)
+        return ByteBuffer.allocate(PREFIX_SIZE_BYTES + 2 * Long.BYTES).order(ByteOrder.BIG_ENDIAN)
```

Comment:
I suggest putting the size into a constant with a description.

Reply:
How will it help? The size is self-evident from the code below, explaining it in the constant would only smear the information

Reply:
I would be grateful (and a little persistent) if you still put the size into a separate constant with a description, it increases the readability and obviousness of the code. At your discretion of course.

Reply:
Ok, I don't mind doing that if you persist